src/package.xml

```diff
@@ -812,6 +812,9 @@
         <members>HH_Container</members>
         <members>HH_ContainerLoadedEvent</members>
         <members>HH_HouseholdSavedEvent</members>
+        <members>HH_keypressEvent</members>
```

Comment:
Our convention for naming module specific aura bundles is MOD_CapitalizedName, so HH_KeypressEvent, etc. would be more in line with our practice.